packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -40,37 +45,24 @@ export const ItemDetailsForm: React.FC<ItemDetailsFormProps> = ({
   allocatedQuantity,
   onChangeItem,
   onChangeQuantity,
-  quantity,
+  // quantity,
   register,
   summaryItem,
-  setPackSize,
-  packSize,
+  packSizeController,
+  availableQuantity,
 }) => {
   const t = useTranslation();
 
-  const quantityDescription = quantity ? (
+  const quantityDescription = allocatedQuantity ? (
```

Comment:
I was unsure about this, but I'm not sure what to do about the allocatedQuantity now?

Reply:
yeah, it doesn't really make sense now. Useful when there is a single quantity and we can show an indication that you are under the requested quantity - but that doesn't apply now.
A summary somewhere might be nice `x units allocated` and/or `x packs of y allocated` 🤷  
Easy enough to show a total number of units allocated, and maybe that's helpful.